1_4.js

```diff
@@ -0,0 +1,24 @@
+let arr = [
+  { "内容": "机を片付ける", "ジャンル": "掃除" },
+  { "内容": "牛乳を買う", "ジャンル": "買い物" },
+  { "内容": "散歩する", "ジャンル": "運動" },
+];
+const printTask = (arr) => {
+  console.log("========================");
+  console.log("現在持っているタスク一覧");
+  console.log("========================");
+  for (let i = 0; i < arr.length; i++) {
+    console.log(`${i} : [内容]${arr[i].内容}、[ジャンル]${arr[i].ジャンル} `);
+  }
+}
+
+printTask(arr);
+
+let enterTask = () => {
```

Comment:
こちらもconstで定義できますね。
関数は定数で定義するものと認識してしまって大丈夫です。
```suggestion
const enterTask = () => {
```